File: src/main/frontend/src/app/models/projectInsightDetailsDTO.ts
import { ProjectInsightProjectDetails } from "./projectInsightDetails";
import { ProjectInsightFormDetails } from "./projectInsightFormDetails";
import { ProjectInsightGroupDetails } from "./projectInsightGroupDetails";
import { ProjectInsightQuestionDetails } from "./projectInsightQuestionDetails";

export class ProjectInsightDetailsDTO {

    projectInsightProjectDetails: ProjectInsightProjectDetails = new ProjectInsightProjectDetails();
    projectInsightGroupDetails: ProjectInsightGroupDetails = new ProjectInsightGroupDetails();
    projectInsightQuestionDetails: ProjectInsightQuestionDetails = new ProjectInsightQuestionDetails();
    projectInsightFormDetails: ProjectInsightFormDetails = new ProjectInsightFormDetails();
}

File: src/main/java/com/apmosys/employeeportal/mongodb/dto/ProjectInsightDetailsDTO.java
package com.apmosys.employeeportal.mongodb.dto;

import com.apmosys.employeeportal.mongodb.modal.ProjectInsightFormDetails;
import com.apmosys.employeeportal.mongodb.modal.ProjectInsightGroupDetails;
import com.apmosys.employeeportal.mongodb.modal.ProjectInsightProjectDetails;
import com.apmosys.employeeportal.mongodb.modal.ProjectInsightQuestionDetails;

import lombok.Data;

@Data
public class ProjectInsightDetailsDTO {

	private ProjectInsightProjectDetails projectInsightProjectDetails;
	private ProjectInsightGroupDetails projectInsightGroupDetails;
	private ProjectInsightQuestionDetails projectInsightQuestionDetails;
	private ProjectInsightFormDetails projectInsightFormDetails;

}
